### PR TITLE
William/rn63 fixes

### PR DESCRIPTION
**Pull request by swansontec**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/components/scenes/FioRequestListScene.js [hunk 1]
```diff
@@ -492,15 +492,15 @@ export class FioRequestList extends React.Component<Props, LocalState> {
...
+      (index > 0 && formatDate(new Date(this.state.fioRequestsPending[index + 1].time_stamp)) !== formatDate(new Date(fioRequest.time_stamp)))
     return <FioRequestRow fioRequest={fioRequest} isLastOfDate={isLastOfDate} onSelect={this.selectPendingRequest} />
   }
 
   renderSent = (itemObj: { item: FioRequest, index: number }) => {
     const { item: fioRequest, index } = itemObj
     const isLastOfDate =
       index + 1 === this.state.fioRequestsSent.length ||
-      (index > 0 && formatDate(new Date(this.state.fioRequestsSent[index + 1].time_stamp), true) !== formatDate(new Date(fioRequest.time_stamp), true))
+      (index > 0 && formatDate(new Date(this.state.fioRequestsSent[index + 1].time_stamp)) !== formatDate(new Date(fioRequest.time_stamp)))
```
Comment (paullinator) [inline src/components/scenes/FioRequestListScene.js:503]: why the change to monthShort false?

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/scenes/FioRequestListScene.js (hunk 1 above)
Comment (swansontec) [reply src/components/scenes/FioRequestListScene.js:503]: This code is trying to see if two things fall on the same day. Rather than using math, they are short-cutting by converting to strings (yuck). So, the comparison ends up working the same with the date being short or long, since `'Tue === Tue'` just the same as `'Tuesday' === 'Tuesday'`, but the longer form seems safer.
